scripts/SMS Sender/README.md

```diff
@@ -0,0 +1,15 @@
+# SMS Sender 
+Easy CLI interface for users looking to test their SMS campaign integration with Kaleyra 
+
+The requirements for any user are :-
+* A kaleyra account and api key, secret key
```

Comment:
Please document how an user can generate their own api key and use it in the script. Use this [documentation](https://github.com/GDSC-RCCIIT/General-Purpose-Scripts/blob/main/scripts/Vaccination-Centre-Locator/README.md) for reference lets say.